MediaBrowser.Providers/Plugins/Tmdb/TmdbExternalUrlProvider.cs

```diff
@@ -53,17 +54,19 @@ public IEnumerable<string> GetExternalUrls(BaseItem item)
                 if (episode.Series.TryGetProviderId(MetadataProvider.Imdb, out seriesExternalId))
                 {
                     var orderString = episode.Series.DisplayOrder;
-                    if (string.IsNullOrEmpty(orderString))
+                    var seasonNumber = episode.Season.IndexNumber;
```

Comment:
```suggestion
                    var seasonNumber = episode.Season?.IndexNumber;
```

I think this was the actual exception from the linked issue